Release 0.10.0

alisawallace opened the pull request:
Release to account for input ID refactors - incremented minor version because changes break dependent test kits.

Jammjammjamm reviewed (approved):
Could you also run `bundle update inferno_core` as part of this?

alisawallace commented:
@Jammjammjamm, FYSA I updated `inferno_core` as you requested and also added FI-3129 Add access token outputs and FI-3156 Loosen scope checks to this release, since those were merged into main before I completed the release.
